src/System.Drawing.Common/src/System/Drawing/Design/CategoryNameCollection.cs

```diff
@@ -3,68 +3,23 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections;
+using System.Security.Permissions;
 
 namespace System.Drawing.Design
 {
-    /// <summary>
```

Comment:
There are other doc comments, but they aren't maintained by the devs on our team, so they are a lot more difficult to change. They also don't show up when you are working on the library itself, which is the main benefit to these comments.